auditbeat/docs/auditbeat-options.asciidoc

```diff
@@ -0,0 +1,62 @@
+//////////////////////////////////////////////////////////////////////////
+//// This content is shared by all Auditbeat modules. Make sure you keep the
+//// descriptions generic enough to work for all modules. To include
+//// this file, use:
+////
+//// include::{docdir}/auditbeat-options.asciidoc[]
+////
+//////////////////////////////////////////////////////////////////////////
+
+[id="module-standard-options-{modulename}"]
+[float]
+==== Standard configuration options
+
+You can specify the following options for any {beatname_uc} module. 
+
+*`module`*:: The name of the module to run.
+
+*`datasets`*:: A list of datasets to execute. Make sure that you only list datasets that
+are available in the module. It is not possible to reference datasets from
+other modules.
+
+*`enabled`*:: A Boolean value that specifies whether the module is enabled. If you use the
+default config file, +{beatname_lc}.yml+, the System module is enabled (set to
```

Comment:
These standard options are included for every module, so do we need to mention the System module explicitly?

Reply:
I think it's worth mentioning at least in the docs for the system module, so I'll make the content conditional.